[2024-04-11T17:32:37.388+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-11T17:32:37.414+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Process_W3_Data.validate_and_clean_data manual__2024-04-11T17:31:57.072023+00:00 [queued]>
[2024-04-11T17:32:37.423+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Process_W3_Data.validate_and_clean_data manual__2024-04-11T17:31:57.072023+00:00 [queued]>
[2024-04-11T17:32:37.423+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-11T17:32:37.440+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): validate_and_clean_data> on 2024-04-11 17:31:57.072023+00:00
[2024-04-11T17:32:37.451+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=23460) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-11T17:32:37.454+0000] {standard_task_runner.py:63} INFO - Started process 23462 to run task
[2024-04-11T17:32:37.454+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Process_W3_Data', 'validate_and_clean_data', 'manual__2024-04-11T17:31:57.072023+00:00', '--job-id', '203', '--raw', '--subdir', 'DAGS_FOLDER/w3c.py', '--cfg-path', '/tmp/tmpugtui3a0']
[2024-04-11T17:32:37.458+0000] {standard_task_runner.py:91} INFO - Job 203: Subtask validate_and_clean_data
[2024-04-11T17:32:37.564+0000] {task_command.py:426} INFO - Running <TaskInstance: Process_W3_Data.validate_and_clean_data manual__2024-04-11T17:31:57.072023+00:00 [running]> on host 426f0b9f9d71
[2024-04-11T17:32:37.792+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Process_W3_Data' AIRFLOW_CTX_TASK_ID='validate_and_clean_data' AIRFLOW_CTX_EXECUTION_DATE='2024-04-11T17:31:57.072023+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-11T17:31:57.072023+00:00'
[2024-04-11T17:32:37.795+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-11T17:32:39.930+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-11T17:32:39.931+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-11T17:32:39.944+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=Process_W3_Data, task_id=validate_and_clean_data, execution_date=20240411T173157, start_date=20240411T173237, end_date=20240411T173239
[2024-04-11T17:32:39.972+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-11T17:32:40.002+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-11T17:32:40.004+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
